faq/sql-faq.md

```diff
@@ -9,11 +9,13 @@ This document summarizes the FAQs related to SQL operations in TiDB.
 
 ## What are the MySQL variables that TiDB is compatible with?
 
-See [The System Variables](/system-variables.md).
+See [System Variables](/system-variables.md).
 
-## Does TiDB support `select for update`?
+## Does TiDB support `SELECT FOR UPDATE`?
 
-Yes. But it differs from MySQL in syntax. As a distributed database, TiDB uses the optimistic lock. `select for update` does not lock data when the transaction is started, but checks conflicts when the transaction is committed. If the check reveals conflicts, the committing transaction rolls back.
+Yes. When using pessimistic locking (the default since TiDB 3.0) it will behave similar to how it does in MySQL.
```

Comment:
```suggestion
Yes. When the pessimistic lock is used (the default lock since TiDB v3.0), the `SELECT FOR UPDATE` execution in TiDB behaves similarly to that in MySQL.
```

Reply:
I believe `optimistic locking` is more correct. `optimistic lock` makes it sound singular: it is the scheme of how all locks will be applied. I will merge the two.